### ISSUE TITLE
Problems with the edit interface for the new assignment builder for editing MCQs

### ISSUE THREAD

**Issue by barbarer** [RunestoneInteractive/rs#516]
In the new assignment builder when you edit a multiple choice question it is not updating the "more" stuff when it is displayed and is not saving several things when you change them.  I am not sure if the problem is that the "more" stuff isn't being saved or just not showing in the edit interface.

**Comment by bnmnetp** [comment]
Yeah, it was getting saved, but I didn't have it connected to show the current values after they were retrieved from the DB.  Fixed, will try to push soon.

**Comment by bnmnetp** [comment]
Closing — the MCQ editor was rebuilt since this was filed. The edit path now hydrates the form from the saved question (`initialData`/`isEdit`), per-option feedback is a first-class field, and there's a separate settings panel for the extras.

If a specific field still fails to round-trip through edit-and-save, please open a new issue naming that field and I'll fix it directly.
